README.md

```diff
@@ -38,21 +38,21 @@ The roadmap and progress to 1.0 can be seen at [issue #2](https://github.com/sam
 
 ## Setting up Federated identity providers
 
-You might have noticed that when adding a Oauth Sign-in methid in the firebase console, you are asked to add a URL that looks something like this to the Oauth's configurations: `https://[app-id].firebaseapp.com/__/auth/handler`
+You might have noticed that when adding a OAuth sign in method in the Firebase console, you are asked to add a URL that looks something like this to the OAuth's configurations: `https://[app-id].firebaseapp.com/__/auth/handler`
 
-What you are essentially doing is whitelisting that URL, which is a hidden URL that exists in every firebase app. When using this library, you will need to add the URL of **your app** instead of the firebase's one. You need to add the URL of the page in your app that will handle the log in. You'll see what I mean in the docs below.
+What you are essentially doing is whitelisting that URL, which is a hidden URL that exists in every firebase app. When using this library, you will need to add the URL of **your app** instead of the Firebase's one. You need to add the URL of the page in your app that will handle the sign in. You'll see what I mean in the docs below.
 
-You might be curious as to why I'm auoiding using firebases endpoint, well, the reasons are:
+You might be curious as to why I'm avoiding using Firebases endpoint, well, the reasons are:
 
-1. It is more secure. The reason you need to whitelist in the first place is for security.
-2. It is way faster, in some cases up to 5 seconds faster.
-3. I don't trust firebase (or anyone) with my user's private data, and you shouldn't either.
+1. More secure, it's the reason you need to whitelist in the first place is for security.
```

Comment:
This fix is not good. The sentence doesn’t make sense. It should be:
`It’s more secure. That’s the reason you need to whitelist in the first place is for security.`

Reply:
resolved in #https://github.com/samuelgozi/firebase-auth-lite/pull/43/commits/86991a59d61e62f433284f7967f3959c1dab546a